TPM-Commands/TPM2_StartAuthSession.md

```diff
@@ -145,6 +165,34 @@ that itself used the EK as its `tpmKey` input.
 > that can be used to satisfy HMAC-based authorization for specific
 > objects.  We will not cover this in detail here.
 
+## Establishing Trust in a TPM
+
+Given a computer that has a discrete TPM, how does software running on
+that computer establish trust in the dTPM?
+
+This is an important question since failure to do this will render the
+computer vulnerable to [certain attacks](https://dolosgroup.io/blog/2021/7/9/from-stolen-laptop-to-inside-the-company-network)
```

Comment:
The issue in the Dolos Group attack is not that the TPM was untrustworthy, but that it exchanged secrets in clear-text.  So this isn't necessarily about "establishing trust" in the TPM, it is about securing the communication with the device. Instead of  "certain attacks" it is probably worth specifying that the system is vulnerable to "a local attacker sniffing sensitive data from the shared TPM bus".

Reply:
Yes, but encrypting to the wrong TPM wouldn't help.

Reply:
That is, just encrypting isn't enough -- the same attack would work if the TPM apps were using encrypted sessions, but did not know a public encryption key or shared `authValue` for the TPM.  You have to know what TPM you're talking to in order to fully defeat this kind of attack, so I addressed the two issues, not just "use encrypted sessions".